File: core/src/main/java/cz/dynawest/jtexy/modules/ParagraphEvent.java

package cz.dynawest.jtexy.modules;

import cz.dynawest.jtexy.parsers.AroundEvent;
import cz.dynawest.jtexy.parsers.TexyParser;

/**
 *
 * @author Ondrej Zizka
 */
public class ParagraphEvent extends AroundEvent {

	public ParagraphEvent(TexyParser parser, String text, TexyModifier modifier) {
		super(parser, text, modifier);
	}

}

File: core/src/main/java/cz/dynawest/jtexy/modules/ParagraphEvent.kt
package cz.dynawest.jtexy.modules

import cz.dynawest.jtexy.events.AroundEvent
import cz.dynawest.jtexy.events.AroundEventListener
import cz.dynawest.jtexy.parsers.TexyParser


class ParagraphEvent(parser: TexyParser, text: String, modifier: TexyModifier?) : AroundEvent(parser, text, modifier)
interface ParagraphEventListener : AroundEventListener<ParagraphEvent>
